gh_actions_exporter/metrics.py

```diff
@@ -11,15 +11,15 @@ class Metrics(object):
     def __init__(self, settings: Settings):
         self.settings = settings
         self.cost = Cost(settings)
-        self.workflow_labelnames = [
+
+        self.common_labelnames = [
             "repository",
             "workflow_name",
             "repository_visibility",
         ]
-        self.job_labelnames = [
-            "repository",
+        self.workflow_labelnames = self.common_labelnames.copy()
+        self.job_labelnames = self.common_labelnames.copy() + [
```

Comment:
For my personal knowledge, any difference between the two?
```suggestion
        self.workflow_labelnames = self.common_labelnames
        self.job_labelnames = self.common_labelnames + [
```

Reply:
If we don't use `copy`, then the variables `workflow_labelnames` and `job_labelnames` will be linked to `common_labelnames` in memory as well. I don't think it's a problem in our case, it's more of a reflex I've developed from working with Java a lot. It can cause confusion. Since this is Python, I think it should be fine without it, but I prefer to keep it.

Reply:
let's keep it, seems like a good call.